### ISSUE TITLE
1.16 Support?

### ISSUE THREAD

**Issue by TheCherry** [maruohon/justenoughdimensions#67]
Is there any chance that we can get a 1.16 supported version?
I need for my MC-Server regular world but in peaceful mode ...

**Comment by maruohon** [comment]
Yes 1.16.5 ports are planned. I still need to do a couple of features for the 1.12.2 version first, and then I'll port it to 1.16.5.
No real ETA, since I also work on a number of other mods and mod ports as well, and I switch between them kinda how I feel like.
